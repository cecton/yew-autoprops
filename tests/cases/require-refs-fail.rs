use yew_autoprops::autoprops_component;

#[autoprops_component]
fn CoolComponent(test: i8, smth: &usize) -> Html {
    println!("test: {}", test);

    html! {
        <div>
            <p>{ smth }</p>
        </div>
    }
}

fn main() {}
