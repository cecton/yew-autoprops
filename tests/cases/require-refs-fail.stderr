error: custom attribute panicked
 --> tests/cases/require-refs-fail.rs:3:1
  |
3 | #[autoprops_component]
  | ^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Invalid argument: test : i8 (must be a reference)
